base-math/trunctdsf.c

```diff
@@ -45,7 +45,7 @@ CONVERT_WRAPPER(
 	a_norm = getmantandexpd128 (a, &exp, 15, 1e15DL);
 	/* Handle obvious overflow and underflow to avoid going beyond the
 	   bounds of the exponent table.  */
-	if (exp > 39)		/* Obvious overflow.  */
+	if (exp > __FLT_MAX_10_EXP__)		/* Obvious overflow.  */
```

Comment:
I think you should use `FLT_MAX_10_EXP`, since this macro is defined in ISO C, as pointed out by Tulio, and you already include `float.h` anyway.

Reply:
@gftg85 Ok I understand that I could use `__FLT_MAX_10_EXP__` or `FLT_MAX_10_EXP`. But, ok I replaced that on my last push. Also changed flt32 to float on the commit message